### DISCUSSION TITLE
Welcome to Torrust Discussions!

### DISCUSSION THREAD

**Discussion by josecelano** [torrust/.github#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BelieveInBunny** [comment]
Nice to meet you
I am Octavian Rex (a.k.a. BelieveInBunny {such a funny username})
I would like to help upgrade the Torrust as a **privacy-focused** along with feature which every regular user wants in their public index.

Thanks
